### ISSUE TITLE
Motr connection configuration to be done via ceph.conf

### ISSUE THREAD

**Issue by andriytk** [andriytk/ceph-old#9]
Currently, it is hard-coded:

```diff
@@ -2262,10 +2264,10 @@ void *newMotrStore(CephContext *cct)
     // XXX: these params should be taken from config settings and
     // cct somehow?
     store->instance = NULL;
-    store->conf.mc_local_addr     = "172.16.179.134@tcp:12345:34:101";
-    store->conf.mc_ha_addr        = "172.16.179.134@tcp:12345:34:1";
-    store->conf.mc_profile        = "0x7000000000000001:0x0";
-    store->conf.mc_process_fid    = "0x7200000000000001:0x0";
+    store->conf.mc_local_addr     = "192.168.180.182@tcp:12345:4:1";
+    store->conf.mc_ha_addr        = "192.168.180.182@tcp:12345:1:1";
+    store->conf.mc_profile        = "0x7000000000000001:0x4f";
+    store->conf.mc_process_fid    = "0x7200000000000001:0x29";
     store->conf.mc_tm_recv_queue_min_len =    64;
     store->conf.mc_max_rpc_msg_size      = 65536;
     store->conf.mc_idx_service_id  = M0_IDX_DIX;
```

It should be done via the ceph.conf file.

**Comment by andriytk** [comment]
Here is a hint to how it can be done:

```diff
diff --git a/src/common/options/rgw.yaml.in b/src/common/options/rgw.yaml.in
index c58d4e8..d65ff55 100644
--- a/src/common/options/rgw.yaml.in
+++ b/src/common/options/rgw.yaml.in
@@ -3245,13 +3245,14 @@ options:
   type: str
   level: advanced
   desc: experimental Option to set backend store type
-  long_desc: defaults to rados. Other valid values are dbstore(experimental).
+  long_desc: defaults to rados. Other valid values are dbstore and motr(experimental).
   default: rados
   services:
   - rgw
   enum_values:
   - rados
   - dbstore
+  - motr
 - name: rgw_luarocks_location
   type: str
   level: advanced
```

This is the diff for motr backend configuration. I suppose it can be done a similar way. See an example of how the configuration is taken at https://github.com/andriytk/ceph/blob/ef8b9c6638f74004ce1ef5c5b26234116eaba2a3/src/rgw/rgw_main.cc#L354

**Comment by andriytk** [comment]
Implemented at commit 4c6ccef.
